Header header
float32 tow
geometry_msgs/Point ecef_pos
geometry_msgs/Vector3 ecef_vel
sensor_msgs/NavSatFix status
